godtools/FlowControllers/BaseFlowController.swift

```diff
@@ -42,4 +43,55 @@ class BaseFlowController: NSObject, UINavigationControllerDelegate {
         navigationController.delegate = self
     }
     
+    // Notifications
+    
+    func defineObservers() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(displayMenu),
+                                               name: .displayMenuNotification,
+                                               object: nil)
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(dismissMenu),
+                                               name: .dismissMenuNotification,
+                                               object: nil)
+    }
+    
+    func displayMenu() {
+        let menuViewController = MenuViewController(nibName: "MenuViewController", bundle: nil)
```

Comment:
One thing I've been doing to create view controllers is using the String.describe initializer. It takes a class which I think will help prevent typos. It does require though that the xib filename match the view controller filename.. which it should anyways. So it's not 100% guarantee, but I think it helps.

In this case:
````swift
let menuViewController = MenuViewController(nibName: String(describing:MenuViewController.self), bundle: nil)
````